components/core/EMail.py

```diff
@@ -1,10 +1,11 @@
 import smtplib
+from ConfReader import conf_reader
 
-username = 'bassa.ucsc@gmail.com'
-password = 'bassa.123'
+conf = conf_reader("email.conf")
 
 def send_mail(toaddrs, msg):
+    conf_reader()
     server = smtplib.SMTP_SSL('smtp.gmail.com:465')
-    server.login(username,password)
```

Comment:
Hmmm if you make the connection to a host, you are the client not the server :33